node/wrappers.js

```diff
@@ -7,34 +7,60 @@ const isObject = require("../lib/utils.js").isObject;
 const utils = require("./utils.js");
 const reifySemVer = utils.getReifySemVer();
 const reifyVersion = reifySemVer.version;
+const hasOwn = Object.prototype.hasOwnProperty;
 
-function addWrapper(func, wrapper) {
-  const reified = func.reified;
-  if (typeof reified.wrappers[reifyVersion] !== "function") {
-    reified.versions.push(reifyVersion);
-    reified.wrappers[reifyVersion] = wrapper;
+function getSymbolForKey(key) {
+  return Symbol.for("reifyWrapperMap$" + key);
```

Comment:
We could make use of a generic `Symbol('__reify')` for entries and this metadata. For this case we can just the `Symbol('__reify')` to hold an object instead of using a symbol per key. That way we try to keep symbol namespace to a minimum and its something unlikely to change.